cli/unregister/unregister.go

```diff
@@ -288,3 +300,39 @@ func RemoveServiceContainers() error {
 		return fmt.Errorf(err_string)
 	}
 }
+
+// backupEventLogs loads eventlogs from eventlog API , marshals them into the JSON format
+// and saves the bkp file into horizon folder with name of backup time
+func backupEventLogs() error {
+	hznDir := "/var/horizon/"
+
```

Comment:
On Mac where anax is running inside a docker container and hzn is running on the host, this directory does not exist. And you may run into the permission problem for different platforms. Please change it to "/tmp/" to be safe.

Reply:
Updated directory to `/tmp/`